app/server/lib/GristServer.ts

```diff
@@ -69,7 +69,7 @@ export interface GristServer {
   servesPlugins(): boolean;
   getBundledWidgets(): ICustomWidget[];
   getBootKey(): string|undefined;
-  getSandboxInfo(): SandboxInfo|undefined;
+  getSandboxInfo(): Promise<SandboxInfo|undefined>;
```

Comment:
Any problems with changing it to the following and making the dummy server throw instead? I don't think `undefined` is returned anymore from the implementation in `FlexServer.ts`.

```suggestion
  getSandboxInfo(): Promise<SandboxInfo>;
```

Reply:
Thanks @georgegevoian, good catch.